### DISCUSSION TITLE
Why doesn't `to_js` work on a Float?

### DISCUSSION THREAD

**Discussion by sebnozzi** [ruby/ruby.wasm#210] [Q&A]
According to the documentation I should be able to call `to_js` in a Float value, right?

https://ruby.github.io/ruby.wasm/Float.html

Then why does this throw an error?

```ruby
require "js"

(Math::PI).to_js
```

Output: 

```
(file): eval:3:in `<main>': undefined method `to_js' for 3.141592653589793:Float (NoMethodError)
 (Exception)
```

Link:
https://try.ruby-lang.org/playground/#code=require+%22js%22%0A%0A(Math%3A%3API).to_js&engine=cruby-3.2.0

**Comment by kateinoigakukun** [comment]
Closed issue https://github.com/ruby/ruby.wasm/issues/212
